### PR TITLE
Spk setup

### PR DESCRIPTION
**Pull request by dennisseah**
this is the first implementation of spk setup command
related to https://github.com/microsoft/bedrock/issues/986

Note:
1. its command decoration json file has a disabled=true property so that the doc generator will skip it.
2. to test this command out, you need to uncomment a liner in index.ts
3. there are common code between spk init and this command; and these common code/functions are moved to lib folder

Next step: create HLD and Manifest Repo

### CODE REVIEW

**Review by andrebriggs — APPROVED** [review]
LGTM

File: src/commands/setup.ts [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+ *
+ * @param coreAPI Core API service
+ * @param name Name of Project
+ */
+export const createProject = async (coreAPI: ICoreApi, name: string) => {
+  try {
+    await coreAPI.queueCreateProject({
+      capabilities: {
+        processTemplate: {
+          templateTypeId: "27450541-8e31-4150-9947-dc59f998fc01" // TOFIX: do not know what this GUID is about
```
Comment (samiyaakhtar) [inline src/commands/setup.ts:138]: Why is this guid needed?

File: src/index.ts [hunk 2]
```diff
@@ -45,6 +46,7 @@ const commandModules = [
         c.version(require("../package.json").version);
       },
       initCommandDecorator
+      // setupCommandDecorator
```
Comment (samiyaakhtar) [inline src/index.ts:49]: is this meant to be commented out?

File: src/commands/setup.ts (hunk 1 above)
Comment (dennisseah) [reply src/commands/setup.ts:138]: this is something that I cannot figure out its value; and I googled for it and found that this value is used. I asked Andre to found a contact where we can ask.

File: src/index.ts (hunk 2 above)
Comment (dennisseah) [reply src/index.ts:49]: yes, this command is not ready hence we are not registering it

Comment (samiyaakhtar) [reply src/index.ts:49]: Got it, thank you

File: src/commands/setup.ts (hunk 1 above)
Comment (samiyaakhtar) [reply src/commands/setup.ts:138]: Got it

**Review by samiyaakhtar — APPROVED** [review]

**Pull request merged**